app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/RowIndividualNameDateOfBirthController.scala

```diff
@@ -78,7 +79,12 @@ class RowIndividualNameDateOfBirthController @Inject() (
     )
 
   private lazy val formsByOrganisationTypes =
-    Seq(CdsOrganisationType.ThirdCountryIndividualId, CdsOrganisationType.ThirdCountrySoleTraderId)
+    Seq(
+      CdsOrganisationType.ThirdCountryIndividualId,
+      CdsOrganisationType.ThirdCountrySoleTraderId,
+      CdsOrganisationType.IsleOfManSoleTraderId,
+      CdsOrganisationType.IsleOfManIndividualId
+    )
```

Comment:
This val seems to share a name with a def in the SixLineAddressController, but in this context has nothing to do with forms but is instead a grouping of Organisations - would overseasOrganisationTypes be a better name?